platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java

```diff
@@ -243,4 +258,12 @@ private URI signSamlGet(
   public void setSystemCrypto(SystemCrypto systemCrypto) {
     this.systemCrypto = systemCrypto;
   }
+
+  public UuidGenerator getUuidGenerator() {
+    return uuidGenerator;
+  }
+
+  public void setUuidGenerator(UuidGenerator uuidGenerator) {
```

Comment:
❓ Since it looks like we only need this to mock out the `UuidGenerator`, could we either make this method package-private and annotate with `@VisibleForTesting`, or inject it in the constructor and get rid of the setter?

Reply:
I changed it to inject the object as a constructor argument and removed the setter.